platform/macos/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 # master
 
+* Fix symbol querying for annotations near tile boundaries at high zoom. ([#12472](https://github.com/mapbox/mapbox-gl-native/issues/12472))
```

Comment:
Replace “symbol querying for annotations” with a reference to “the `-[MGLMapView annotationAtPoint:]` method”.